DeadmansSwitch.sol

```diff
@@ -4,33 +4,49 @@ pragma solidity ^0.8.0;
 contract DeadmansSwitch {
     // TODO: Declare state variables
     // Hint: You'll need variables for the owner, beneficiary, and last check-in block
+    address owner;
+    address beneficiary;
+    uint check_in_block;
 
     // TODO: Implement constructor
     constructor(address _beneficiary) {
         // Hint: Initialize state variables
+        owner=msg.sender;
+        beneficiary=_beneficiary;
     }
 
     // TODO: Implement still_alive function
     function still_alive() public {
         // Hint: Update the last check-in block
+        check_in_block=block.number;
     }
 
     // TODO: Implement release funds function
-    function releaseFunds() public {
+    function releaseFunds() public payable{
         // Hint: Check if 10 blocks have passed since last check-in
+        require(block.number>=10+check_in_block, "10 blocks has not passed");
```

Comment:
require condition is not correct, as you have to check for 10th block also, if it has passed or not.